src/toast/schedule.py

```diff
@@ -337,6 +295,20 @@ def _parse_line(line):
             except Exception:
                 start_time = dateutil.parser.parse(start_time)
                 stop_time = dateutil.parser.parse(stop_time)
+            # DEBUG begin
+            print(f"PARSED ground scan with \n"
+                  f"PARSED name = {name} \n"
+                  f"PARSED start_time = {start_time} \n"
+                  f"PARSED stop_time = {stop_time} \n"
+                  f"PARSED boresight_angle = {boresight_angle} \n"
+                  f"PARSED azmin = {azmin} \n"
+                  f"PARSED azmax = {azmax} \n"
+                  f"PARSED el = {el} \n"
+                  f"PARSED scan = {scan} \n"
+                  f"PARSED subscan = {subscan} \n",
+                  flush=True,
+            )
+            # DEBUG end
```

Comment:
Did you intend to leave this here?  Maybe concatenate to a string and use `log.debug()`?

Reply:
I didn't mean to leave it. Thanks for noticing!